### PR TITLE
Updates README

### PR DESCRIPTION
**Pull request by petems**
Adds instructor tag for me! :smile:

### CODE REVIEW

File: doc/authors/README.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Peter Souter
+Peter Souter - Instructor
```
Comment (petems) [inline doc/authors/README.txt]: LGTM!

Comment (petems) [inline doc/authors/README.txt]: :+1:

**Comment by jexley** [comment]
Looks good to me too!

File: doc/authors/README.txt (hunk 1 above)
Comment (oriolntt) [inline doc/authors/README.txt]: LGTM!

**Pull request merged**